main.js

```diff
@@ -383,13 +383,39 @@ function sortsch(sch){
 	sch.sort(function(a, b){if(b.d < a.d || (b.d == a.d && b.s < a.s)) return 1; return -1;});
 	return sch;
 }
+
+function calc_sleep_score(sch){
+	var sleep_score = 0;
+	for(var i=0; i < sch.length; i++){
+		if(sch[i].p != undefined){
```

Comment:
Does this ever happen? Also `null == undefined` in javascript, you might want to use `!==`. (Like in good old php...)

Reply:
Actually, this line ignores the Don't Fill blocks. But you are right. I'll change with `!==`.